livedev/livedev

```diff
@@ -62,15 +63,19 @@ def parse_checksum_output(output, relative=None):
     return data
 
 class Remote(object):
-    def __init__(self, host, workers=1, verbose=False):
+    def __init__(self, host, workers=1, verbose=False, ssh_copy_id=False):
         # host, path = rpath.split(':', 1)
         # self.rpath = rpath
         self.host = host
         # self.path = path
         self.workers = workers
         self.verbose = verbose
+        self.ssh_copy_id = ssh_copy_id
         self.manager = None
 
+        if self.ssh_copy_id:
+           self.copy_ssh_id()
```

Comment:
I would prefer avoiding operations in the constructor of the Remote object this is bad style.
Could you move this logic in the main function before `if args.init` line 411

This way there is no need to add a new attribute in the class that doesn't matter after init.